### PR TITLE
Styling changes: modals, notifications, overall colors

### PR DESCRIPTION
**Pull request by mjadach-iv**

### CODE REVIEW

File: src/future-hopr-lib-components/Layout/drawer.tsx [hunk 1]
```diff
@@ -23,8 +23,8 @@ const StyledDrawer = styled(MuiDrawer)`
     padding-top: 59px;
     transition: width 0.4s ease-out;
     overflow-x: hidden;
-    background: #3c64a5;
-    color: white;
+    /* background: #3c64a5;
+    color: white; */
```
Comment (ItsLuized) [inline src/future-hopr-lib-components/Layout/drawer.tsx:27]: we should remove the comments on the styled components.

**Review by ItsLuized — CHANGES_REQUESTED** [review]
I found the issue with the scroll bar ( I think ) happening in the metamask menu when it's not connected.

A white line appears to the side of the right drawer
image

File: src/future-hopr-lib-components/Layout/drawer.tsx (hunk 1 above)
Comment (mjadach-iv) [reply src/future-hopr-lib-components/Layout/drawer.tsx]: Left it as I wanted to re-use it in next pr for the Staking Hub drawer. It's fixed now

**Comment by mjadach-iv** [comment]
> I found the issue with the scroll bar ( I think ) happening in the metamask menu when it's not connected.
> 
> A white line appears to the side of the right drawer image

fixed

**Review by ItsLuized — APPROVED** [review]

**Pull request merged**